packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -1,24 +1,33 @@
 import { useFeature } from '../components/GrowthBookProvider';
 import { feature } from '../lib/featureManagement';
 import { FeedLayout } from '../lib/featureValues';
+import { AllFeedPages } from '../lib/query';
+import { SharedFeedPage } from '../components/utilities';
 
 interface UseFeedLayoutProps {
-  feedName: string;
+  feedName: AllFeedPages;
+}
+
+interface UseFeedLayoutReturn {
+  isSingleColumnFeedLayout: boolean;
 }
 
-// TODO: return type to be changed once more info is added to the return object
 export const useFeedLayout = ({
   feedName,
-}: UseFeedLayoutProps): Record<string, unknown> => {
+}: UseFeedLayoutProps): UseFeedLayoutReturn => {
   const feedLayoutVersion = useFeature(feature.feedLayout);
   const isFeedLayoutVersion = feedLayoutVersion === FeedLayout.V1;
-  const shouldUseFeedLayout = isFeedLayoutVersion && feedName !== 'squad';
+  const isIncludedFeed = Object.values(SharedFeedPage).includes(
+    feedName as SharedFeedPage,
+  );
+  const shouldUseFeedLayout = isFeedLayoutVersion && isIncludedFeed;
```

Comment:
The name is confusing as everything is a feed layout no?
You want to evaluate if something is v1 and applicable?

Reply:
Yeah true, would this be ok `shouldUseFeedLayoutV1` ?